include/simdjson/generic/ondemand/parser-inl.h

```diff
@@ -82,10 +87,20 @@ simdjson_warn_unused simdjson_really_inline simdjson_result<json_iterator> parse
 simdjson_really_inline size_t parser::capacity() const noexcept {
   return _capacity;
 }
+simdjson_really_inline size_t parser::max_capacity() const noexcept {
+  return _max_capacity;
+}
 simdjson_really_inline size_t parser::max_depth() const noexcept {
   return _max_depth;
 }
 
+simdjson_really_inline void parser::set_max_capacity(size_t max_capacity) noexcept {
+  if(max_capacity < MINIMAL_DOCUMENT_CAPACITY) {
```

Comment:
@NicolasJiaxin This causes some build issues. I recommend you duplicate the variable,

```
size_t MINIMAL_ONDEMAND_DOCUMENT_CAPACITY  = 32;
```

In the current scope.